src/server/getters/get-orders.ts

```diff
@@ -31,6 +31,8 @@ export function getOrders(db: Knex, universe: Address|null, marketID: Address|nu
   query.leftJoin("blocks", "orders.blockNumber", "blocks.blockNumber");
   query.leftJoin("markets", "orders.marketID", "markets.marketID");
   query.where(queryData);
+  if (earliestCreationTime != null) query.where("creationTime", ">", earliestCreationTime);
```

Comment:
@bthaile thoughts about inclusiveness?

Reply:
that was the route I was going to go. should we include the time passed in so >= ???